src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestResult.cs

```diff
@@ -553,17 +553,6 @@ internal void AddCollectorDataEntries(IEnumerable<CollectorDataEntry> collectorD
             {
                 Debug.Assert(collectorDataEntry != null, "'collectorDataEntry' is null");
                 Debug.Assert(!this.collectorDataEntries.Contains(collectorDataEntry), "The collector data entry already exists in the collection");
-#if DEBUG
-                // Verify that any URI data attachments in the entry have relative paths
-                foreach (IDataAttachment attachment in collectorDataEntry.Attachments)
-                {
-                    UriDataAttachment uriDataAttachment = attachment as UriDataAttachment;
-                    if (uriDataAttachment != null)
-                    {
-                        Debug.Assert(uriDataAttachment.Uri.IsAbsoluteUri, "'collectorDataEntry' contains a URI data attachment with a relative URI");
```

Comment:
Same goes here, if can pick attachments, from relative URI, the change it that this path should not be null/empty